Mutagen.Bethesda.Analyzers.Skyrim/Extensions/FactionExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Mutagen.Bethesda.Skyrim;
+namespace Mutagen.Bethesda.Analyzers.Skyrim;
+
+public static class FactionExtensions
+{
+    public static bool IsVendor(this IFactionGetter faction)
+    {
+        return (faction.Flags & Faction.FactionFlag.Vendor) != 0;
```

Comment:
We should use HasFlag
`return faction.Flags.HasFlag(Faction.FactionFlag.Vendor);`

Reply:
I know we have `HasFlag` but I thought for extension utility calls like this it would be preferred to use the faster bitwise operation? Probably doesn't matter very much but as this can be used a lot in various scenarios it might be worth sticking to this?

Reply:
Fair.  I'll see if i can add some extension function to get the best of both